src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.log4j.Logger;
 import org.openqa.selenium.By;
 import org.openqa.selenium.ElementNotVisibleException;
+import org.openqa.selenium.NoSuchSessionException;
```

Comment:
This class was just added in 2.53.0 (if I checked the version correctly) but the DOM XSS add-on is targeting all versions of the Selenium add-on which have versions older than that (e.g. 2.45.0), the manifest should be updated to require a newer version of the Selenium add-on (7?).